packages/core/test/controls/numericInputTests.tsx

```diff
@@ -212,6 +218,410 @@ describe("<NumericInput>", () => {
         });
     });
 
+    describe("Keyboard text entry in input field", () => {
+
```

Comment:
i think you can pull a bunch of the repeating constants to the top, especially the keys